src/krux/pages/__init__.py

```diff
@@ -235,7 +325,7 @@ def print_qr_prompt(self, data, qr_format):
         time.sleep_ms(1000)
         self.ctx.display.draw_centered_text(t("Print to QR?"))
         btn = self.ctx.input.wait_for_button()
-        if btn == BUTTON_ENTER:
+        if btn in (BUTTON_ENTER, BUTTON_TOUCH):
```

Comment:
Can we tweak this so that there is a function in `input.py` where we can pass in a button and get back a boolean instead?

Something like:
```
def is_proceed_button(btn):
     return btn in (BUTTON_ENTER, BUTTON_TOUCH)
```
Then all these places could become:
```
if is_proceed_button(self.ctx.input.wait_for_button()):
```

This way we don't accidentally forget to put one of these buttons in a condition, with the added bonus of DRYing up the code.